bait-files/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -34,16 +32,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
     UserService userService = UserServiceFactory.getUserService();
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-    Entity newUser = new Entity("User", request.getParameter("username"));
-    newUser.setProperty("id", userService.getCurrentUser().getUserId());
-    newUser.setProperty("username", request.getParameter("username"));
-    newUser.setProperty("first-name", request.getParameter("first-name"));
-    newUser.setProperty("last-name", request.getParameter("last-name"));
-    newUser.setProperty("is-admin", false);
+    String userId = userService.getCurrentUser().getUserId();
+    String username = request.getParameter("username");
+    String firstName = request.getParameter("first-name");
+    String lastName = request.getParameter("last-name");
```

Comment:
Let's make these strings constants somewhere. I know Iman had some constants too. You all could just make a Constants.java

Reply:
Added constants